diff --git a/staging/src/k8s.io/apiserver/pkg/storage/etcd3/testserver/test_server.go b/staging/src/k8s.io/apiserver/pkg/storage/etcd3/testserver/test_server.go
index b8c5641c069..aa78b9a3da6 100644
--- a/staging/src/k8s.io/apiserver/pkg/storage/etcd3/testserver/test_server.go
+++ b/staging/src/k8s.io/apiserver/pkg/storage/etcd3/testserver/test_server.go
@@ -119,9 +119,13 @@ func RunEtcd(t testing.TB, cfg *embed.Config) *kubernetes.Client {
 		t.Fatal(err)
 	}
 
+	tlsConfig.InsecureSkipVerify = true
+
 	client, err := kubernetes.New(clientv3.Config{
 		TLS:         tlsConfig,
-		Endpoints:   e.Server.Cluster().ClientURLs(),
+		Endpoints:   []string{"https://localhost:2379"},
+		Username:    t.Name(),
+		Password:    t.Name(),
 		DialTimeout: 10 * time.Second,
 		DialOptions: []grpc.DialOption{grpc.WithBlock()},
 		Logger:      zaptest.NewLogger(t, zaptest.Level(zapcore.ErrorLevel)).Named("etcd-client"),
